stdlib/Artifacts/src/Artifacts.jl

```diff
@@ -418,7 +418,7 @@ collapsed artifact.  Returns `nothing` if no mapping can be found.
 """
 function artifact_hash(name::String, artifacts_toml::String;
                        platform::AbstractPlatform = HostPlatform(),
-                       pkg_uuid::Union{Base.UUID,Nothing}=nothing)
+                       pkg_uuid::Union{Base.UUID,Nothing}=nothing)::SHA1
```

Comment:
This can also return `nothing`, no?

Reply:
Ouch... I added that as an afterthought just before pushing. Let's see if the test catch it ;)

Reply:
It didn't..